sogs/routes/rooms.py

```diff
@@ -278,6 +278,24 @@ def update_room(room):
     return jsonify({})
 
 
+@rooms.get("/room/<Room:room>/permInfo")
+@auth.mod_required
+def get_permission_info(room):
+    """
+    Fetches permissions about the room, like ban info etc.
+
+    # Query Parameters
+
+    TODO: implement me
+
+    # Return Value
+
+    dict of session_id to current permission info
+
+    """
+    return jsonify(room.export_permissions(g.user))
```

Comment:
I'm worried this might get big, as is, because it'll be something like:
```json
{"15abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789": {"read": true, "upload": false},"15bcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789a": {"banned": true},
...}
```

It might be worth encoding it to something smaller, perhaps by instead of a dict of "perm": true|false, we could return a set (I dunno if jsonify is happy converting a set to a list; might need to be done manually) containing:
"r"/"-r" - included if read is not null
"w"/"-w" included if write is not null
"a"/"-a" included if accessible is not null
"u"/"-u" included if upload is not null

Some of the flags are non-nullable (because they are additive rather than overriding) and so wouldn't have "-" variations, i.e. would only be included when true:
"b" would be included only if banned is true
"A" would be included only if admin is true
"M" would be included only if moderator is true *and* admin is not true
"V" would be included only if moderator is true and visible_mod is true